src/components/crossDevice/MobileConnected/index.tsx

```diff
@@ -1,12 +1,20 @@
-import { h } from 'preact'
+import { h, FunctionComponent } from 'preact'
 import PageTitle from '../../PageTitle'
 import { trackComponent } from '../../../Tracker'
 import { preventDefaultOnClick } from '~utils/index'
 import { localised } from '../../../locales'
 import theme from '../../Theme/style.scss'
 import style from './style.scss'
 
-const MobileConnected = ({ translate, back }) => (
+import type { WithLocalisedProps, WithTrackingProps } from '~types/hocs'
+
+type MobileConnectedProps = {
+  back: () => void
+}
+
+type Props = MobileConnectedProps & WithLocalisedProps & WithTrackingProps
+
+const MobileConnected: FunctionComponent<Props> = ({ translate, back }) => (
```

Comment:
You can use `useLocales()` hook here to avoid passing `translate` prop (and can eliminate `WithLocalisedProps` too) :)